tests/profile.test.js

```diff
@@ -0,0 +1,277 @@
+import '@babel/polyfill';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import status from '../src/helpers/constants/status.codes';
+import server from '../src/index';
+import signup from '../src/helpers/tests/signup';
+import errors from '../src/helpers/constants/error.messages';
+
+chai.use(chaiHttp);
+chai.should();
+
+
+const data = {
+  country: 'congo',
+  firstName: 'alain',
+  lastName: 'burindi',
+  gender: 'man',
+  phoneNumber: '+243974690917',
+  address: 'gachinjiro/kigali',
+  profileImage: 'noimage.jpg',
+  bio: 'I work at statefarm',
+  facebook: 'https://facebook.com/alainburindi',
+  twitter: 'https://twitter.com/alainburindi',
+};
+
+// const errors = {
+//   country: 'The country is required should only contain alphanumeric characters',
+//   firstName: 'The firstName should only contain alphanumeric characters',
+//   lastName: 'The lastName should only contain alphanumeric characters',
+//   address: 'The address is required',
+//   gender: 'The gender should only contain alphanumeric characters',
+//   phoneNumber: 'The phone number is required',
+//   bio: 'The bio is required',
+//   user: 'User doesn\'t exist',
+// };
+
+const {
+  country, firstName, lastName,
+  address, gender, profileImage, phoneNumber, bio
+} = data;
+
+const authToken = signup();
```

Comment:
@alainburindi please, inform your team-mates so they can use this signup helper method as well.